src/App/index.jsx

```diff
@@ -6,7 +6,9 @@ import {
 } from 'react-router-dom';
 import Benchmark from '../views/Benchmark';
 import Navigation from '../components/Navigation';
-import { CONFIG } from '../config';
+// import { CONFIG } from '../config';
+import DEFAULT_DESKTOP_BENCHMARKS from '../configuration/js-team/appDefaults';
```

Comment:
I *think* you can do the following:

```suggestion
import { landingPath } from '../configuration/js-team/appDefaults';
```

Reply:
> I _think_ you can do the following:
> 
> span class="pl-k"> import DEFAULT_DESKTOP_BENCHMARKS from '../configuration/js-team/appDefaults';
> span class="pl-k"> import { landingPath } from '../configuration/js-team/appDefaults';

**It is not allowed to destructure a default export while importing. ([reference](https://stackoverflow.com/questions/43814830/destructuring-a-default-export-object/43987935#43987935))**

One way to achieve destructuring while import is below:

in appDefault.js

```
export const landingPath = '/win10/overview?numDays=90';
export const timeRange = 90; // 90 days
export const colors = ['#e55525', '#ffcd02'];
export const labels = ['Firefox', 'Chrome'];

const DASHBOARD_CONFIG = {
  landingPath,
  timeRange,
  colors,
  labels,
};

export default DASHBOARD_CONFIG;
```

in index.js

`import { landingPath } from '../configuration/js-team/appDefaults';`
It is working this way. What are your thoughts on it?

Reply:
What you had at the beginning is fine. Thanks for checking!

Move appDefaults to `src/configuration`.